MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -242,7 +244,9 @@ - (instancetype)init {
     _logBufferDir = [MSCrashesUtil logBufferDir];
     _analyzerInProgressFile = [_crashesDir URLByAppendingPathComponent:kMSAnalyzerFilename];
     _didCrashInLastSession = NO;
+#if !TARGET_OS_TV
     _enableMachExceptionHandler = YES;
+#endif
```

Comment:
That was my first implementation but we cannot eliminate the macro at line 525.
`PLCrashReporterSignalHandlerTypeMach` is not available on tvOS SDK so anyway we have to do this one.